docs/concepts/personal-access-token.mdx

```diff
@@ -47,6 +47,42 @@ secure and rotate them if necessary.
 
 :::
 
+## Authorizing Ory Network Workspace Admin APIs
```

Comment:
It totally makes sense to have it in the API section of the docs but I would also consider linking it into new section in the docs for Ory Console as you have to use console to create WAK or AK.